corehq/apps/sms/change_publishers.py

```diff
@@ -1,10 +1,20 @@
 from corehq.apps.change_feed import topics
```

Comment:
for the tests - i had seen a bunch of errors in the travis logs about unrelated sms tests not being able to publish to kafka. it was making it difficult to debug actual test failures so i stopped the publish altogether in tests unless it's needed.  do we not expect any kafka errors to be raised in non-pillow tests?